youtube_dl/extractor/adobepass.py

```diff
@@ -325,6 +320,11 @@
     'col080': {
         'name': 'Columbus Telephone'
     },
+    'Comcast_SSO': {
+        'name': 'Comcast XFINITY',
+        'username_field': 'user',
+        'password_field': 'passwd',
+    },
```

Comment:
the list of MSOs that depend on watchTVeverywhere was automatically generated(3a5a18705f2a7faf64a4b69665511ef5f0c6084d) and keeping it grouped in the bottom of the list was intentional.